services/painting-surface-data-simulator-service/app/routers/simulator_router.py

```
@@ -0,0 +1,77 @@
+                try:
+                    logs.append(json.loads(line.strip()))
+                except:
+                    continue
+
+        return {
+            "logs": logs,
+            "total_count": len(lines) if 'lines' in locals() else 0
+        }
+
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"로그 조회 실패: {str(e)}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)